pkg/e2e/operators/cloudingress/deletetapps2.go

```diff
@@ -0,0 +1,146 @@
+package cloudingress
+
+import (
+	"context"
+	"log"
+	"strings"
+
+	"time"
+
+	"github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	cloudingressv1alpha1 "github.com/openshift/cloud-ingress-operator/pkg/apis/cloudingress/v1alpha1"
+	"github.com/openshift/osde2e/pkg/common/constants"
+	"github.com/openshift/osde2e/pkg/common/helper"
+	"github.com/openshift/osde2e/pkg/e2e/operators"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var _ = ginkgo.Describe(constants.SuiteInforming+TestPrefix, func() {
+	h := helper.New()
+	ginkgo.Context("Delete apps2 ingresscontroller", func() {
+		ginkgo.It("Should not be able to delete the ApplicationIngress that doesn't belong to CIO", func() {
+			//1. create a secondaryIngress that does belong to cloud-ingress-operator
+			secondaryIngress := secondaryIngress(h)
+
+			//only create the secondary ingress if it doesn't exist already in the publishing strategy
+			if _, exists, _ := appIngressExits(h, false, secondaryIngress.DNSName); !exists {
+				addAppIngress(h, secondaryIngress)
```

Comment:
I would add a wait after this. CIO might take some time to create the ingresscontroller.